.github/workflows/_integration_tests.yml

```diff
@@ -128,6 +130,15 @@ jobs:
       - uses: ./.github/actions/ghcr-docker-login
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Check minimum client version
+        id: version_check
+        if: ${{ matrix.test.min_client_version }}
+        continue-on-error: true
+        run: |
+          ACTUAL_VERSION=$(docker run ${{ inputs.client_image }}:${{ inputs.client_tag }} firezone-headless-client --version | awk '{print $2}')
+          MIN_VERSION="${{ matrix.test.min_client_version }}"
+
+          [ "$(printf '%s\n' "$MIN_VERSION" "$ACTUAL_VERSION" | sort --version-sort | head -n1)" == "$MIN_VERSION" ]
```

Comment:
I added this so we can have this test in here even though it does not pass the compatibility tests (see https://github.com/firezone/firezone/actions/runs/18023609619?pr=10462). With this check in here, we can have the integration test run for the current build of the client and it will also start running as soon as we make a release.